stbt_rig.py

```diff
@@ -882,19 +882,14 @@ def __init__(self, root=None, remote="origin"):
         self.root = root
         self.remote = remote
 
-    def _git(self, cmd, capture_output=True, extra_env=None, **kwargs):  # pylint:disable=no-self-use
-        if capture_output:
-            call = subprocess.check_output
-        else:
-            call = subprocess.check_call
-
+    def _git(self, cmd, extra_env=None, **kwargs):  # pylint:disable=no-self-use
```

Comment:
> Refactor: Remove unused `capture_output` argument 

I think this should be set to `False` for our call to git push during the snapshot (at least in interactive mode)?  This way people can enter passwords.

Reply:
Of course this isn't a regression so let's not worry about it for now. It can be reintroduced in another PR.